screens/Events/Events.component.js

```diff
@@ -11,15 +11,17 @@ const { width } = Dimensions.get('window');
 
 const cardItems = [
     {
-        title: "Career Center Workshop",
+        title: "Career Center Work",
```

Comment:
Not a big deal, but isn't it a workshop instead of work?

Reply:
Yeah my mistake.